lib/pronto.rb

```diff
@@ -29,6 +29,9 @@
 require 'pronto/formatter/colorizable'
 require 'pronto/formatter/text_formatter'
 require 'pronto/formatter/json_formatter'
+require 'pronto/formatter/git_formatter_base'
```

Comment:
Actually, I suggest removing `base` from the name too. Not using that convention anywhere else in this project.